lib/feature/home_page/home_page_connector.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:counter_async_redux/feature/home_page/home_page.dart';
```

Comment:
why is your package still named `counter_async_redux`?

Reply:
Copied dependencies. Will change.

Reply:
pubspec.yaml *